recipes/icon-dsl/25.8/g200/environments.yaml

```diff
@@ -0,0 +1,11 @@
+myenv:
+  compiler:
+  - toolchain: gcc
+    spec: gcc
+  specs:
+  - icon4py@icon_20250328 +distributed +cuda
+  - py-mpi4py
+  - python
+  unify: true
```

Comment:
Think about adding variants, so that any cuda code is compiled correctly.

Also, I notice that you are installing rust and rust bootstrap, in order to install `uv`. This is a lot of overhead for something that can be downloaded as a stand alone binary.